src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/index.spec.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { renderImageSnapshot } from 'utils/tests/imageSnapshot';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+import { converse } from 'utils/window';
+import { InputDisplayNameOverlay } from './index';
+
+const mockSetInputBarActive = jest.fn();
+const mockSetOverlay = jest.fn();
+
+jest.mock('utils/window', () => ({
+  converse: {
+    claimNewNicknameInChatRoom: jest.fn(),
+  },
+}));
+
+describe('<InputDisplayNameOverlay />', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("controls input and show error when input contains 'Anonymous'", () => {
```

Comment:
```suggestion
  it("controls input and shows error when input contains 'Anonymous'", () => {
```

Reply:
done in commit [d3c2a0](https://github.com/openfun/marsha/pull/1276/commits/d3c2a037d1f2b70d6fd332d3c8951c3bea0df3a5)